client/src/components/AuthProvider.ts

```diff
@@ -202,11 +202,15 @@ export class SASAuthProvider implements AuthenticationProvider, Disposable {
   private async getStoredSessions(): Promise<
     Record<string, SASAuthSession> | undefined
   > {
-    const storedSessionData = await this.secretStorage.get(SECRET_KEY);
-    if (!storedSessionData) {
+    try {
```

Comment:
My 2 cents is that we should not hide this error. If user don't know this error, they have no idea why their stored session doesn't work. And will think it's a bug of the extension. The real solution is to clean their password.
I've documented it in our wiki.
https://github.com/sassoftware/vscode-sas-extension/wiki/FAQ#i-got-unable-to-parse-decrypted-password-error